src/main/resources/application.conf

```diff
@@ -31,6 +31,15 @@ authorizer {
     "zookeeper.session.timeout.ms" = ${?AUTHORIZER_SESSION_TIMEOUT_MS}
     "zookeeper.set.acl" = "false"
     "zookeeper.set.acl" = ${?AUTHORIZER_ZOOKEEPER_SET_ACL}
+    // options below needed when AUTHORIZER_CLASS is "com.github.simplesteph.ksm.compat.AdminClientAuthorizer"
+    // see org.apache.kafka.clients.admin.AdminClientConfig for more options
+    "client.id"         = "kafka-security-manager"
```

Comment:
my understanding is that any properties passed here would be going directly into the authorizer right? I'm slightly annoyed by the fact they're mixed with zookeeper. 

Maybe we can have a `zookeeper{}` configuration block and a `kafka{}` configuration block under the authorizer? Although that wouldn't be backwards compatible

Reply:
Agree, that's also the place I'm not happy with :) If breaking compatibility and version bump would be ok, I'll suggest a safer approach

Reply:
If so, could you please tag current version as `0.7` so I'll request `0.8-snapshot` in PR?

Reply:
go ahead with this version it's no worries

Reply:
let's keep 0.7-SNAPSHOT.
Overall I think we mildly break the compatibility as we can keep the environment variables named the same.
Please include the most important settings as env variables (optionals) and only include the non null ones in the Admin client

`ssl.key.password`
`ssl.keystore.location`
`ssl.keystore.password`
`ssl.truststore.location`
`ssl.truststore.password`
`sasl.jaas.config`
`sasl.mechanism`
`security.protocol`

Also please document the ACL Kafka Security Manager needs to manage Kafka. There's also a chance users lock themselves out if they remove that needed ACL for Kafka Security Manager.

Reply:
Including these properties only for Admin client in ksm code may break compatibility for customized AUTHORIZER_CLASS & overriden `application.conf` if they rely on these properties. I think it would be better to put these properties into a different section (like `authorizer.admin-client-config` so it won't break anything.